src/SFML/Graphics/Text.cpp

```diff
@@ -380,6 +380,70 @@ Vector2f Text::findCharacterPos(std::size_t index) const
 }
 
 
+////////////////////////////////////////////////////////////
+bool Text::fitsWithinBounds(const sf::FloatRect& bounds, std::size_t& oob) const
+{
+    // Make sure that we have a valid font
+    if (!m_font)
+    {
+        oob = m_string.getSize();
+
+        return true;
+    }
+
+    // Precompute the variables needed by the algorithm
+    bool  isBold          = m_style & Bold;
```

Comment:
Could this be `const bool` instead, doesn't appear to be changing after this line so I'd guess so.

Reply:
Done :)